artemis/modules/domain_expiration_scanner.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+import datetime
+import time
+from typing import Any, Dict
+
+from karton.core import Task
+from whois import Domain, WhoisQuotaExceeded, query  # type: ignore
+
+from artemis.binds import TaskStatus, TaskType
+from artemis.config import Config
+from artemis.domains import is_main_domain
+from artemis.module_base import ArtemisBase
+
+
+class DomainExpirationScanner(ArtemisBase):
+    """
+    Alerts if domain expiration date is coming.
+    """
+
+    identity = "domain_expiration_scanner"
+    filters = [{"type": TaskType.DOMAIN.value}]
+
+    def run(self, current_task: Task) -> None:
+        domain = current_task.get_payload(TaskType.DOMAIN)
+        result: Dict[str, Any] = {}
+        status = TaskStatus.OK
+        status_reason = None
+        if is_main_domain(domain):
+            try:
+                domain_data = self._query_whois(domain=domain)
+                expiry_date = domain_data.expiration_date
+                result = self._prepare_expiration_data(expiration_date=expiry_date, result=result)
+                status = TaskStatus.INTERESTING
```

Comment:
now you set TaskStatus.INTERESTING always, not only if the domain is close to expiry date